pkg/server/container_start.go

```diff
@@ -176,7 +177,12 @@ func resetContainerStarting(container containerstore.Container) error {
 
 // createContainerLoggers creates container loggers and return write closer for stdout and stderr.
 func (c *criService) createContainerLoggers(logPath string, tty bool) (stdout io.WriteCloser, stderr io.WriteCloser, err error) {
-	if logPath != "" {
+	u, err := url.Parse(logPath)
+
+	if logPath == "" || (err == nil && u.Scheme == "binary") {
+		stdout = cio.NewDiscardLogger()
+		stderr = cio.NewDiscardLogger()
```

Comment:
What happens if there is an error parsing or the schema isn't binary but the logpath is still == ""? Would falling into the else statement be the expected route?

Reply:
> What happens if there is an error parsing or the schema isn't binary but the logpath is still == ""?

in this case `logPath  == ""` should eval first and second statement shouldn't have any effect.

Accepting only `"binary"` as scheme and return an error otherwise makes sense, that's what we ended up doing on hcsshim side anyways.